### PR TITLE
ExaWorks theme and tutorials cleanup

### PR DESCRIPTION
**Pull request by mturilli**
This switches the documentation theme to the one used for PSIJ. It also cleans up the tutorials by:
- Removing redundant sub directories 
- Fixing the headings hierarchy across all tutorials
- Generating a toctree in the tutorial's page
- Moving and renaming support files into `images`

Note:
- Parsl tutorial is missing its data file that should be uploaded into `tutorials/data`
- Parsl tutorial seems to be failing to create a needed directory before cd into it

The notes mentioned above should not block the merging of this branch as they will be addressed by a separate PR.

### CODE REVIEW

File: docs/source/tutorials.rst [hunk 1]
```diff
@@ -40,12 +40,19 @@ To run the tutorials in the ExaWorks SDK Docker container:
...
-* `Swift/T <tutorials/swift-t/swift-t.ipynb>`_
+
+SDK Tutorials
+-------------
+
+.. * `Flux <tutorials/flux.ipynb>`_
+.. * `Parsl <tutorials/parsl.ipynb>`_
+.. * `PSI/J <tutorials/psij.ipynb>`_
+.. * `RADICAL-Cybertools <tutorials/rct.ipynb>`_
+.. * `Swift/T <tutorials/swift.ipynb>`_
```
Comment (mtitov) [inline docs/source/tutorials.rst:51]: Regarding subdirectories for Parsl and Swift/T:
- Parsl: In a corresponding docker container, we do load an additional tutorial repo, thus it is good to have a dedicated directory, where to keep it (here is a [link](https://github.com/ExaWorks/SDK/blob/8e112326b841705cf5ba60cbc18152de8e5ccd8b/docker/tutorials/Dockerfile#L46-L47) from PR #162 )
- Swift/T: Justin's comment regarding this - https://github.com/ExaWorks/SDK/pull/161#issuecomment-1409576597

File: docs/source/tutorials/parsl-rp.ipynb [hunk 2]
```diff
@@ -304,7 +297,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.10"
+   "version": "3.11.1"
```
Comment (mtitov) [inline docs/source/tutorials/parsl-rp.ipynb:300]: should we keep PY version any from 3.7, 3.8, 3.9?

File: docs/source/tutorials/parsl.ipynb [hunk 3]
```diff
@@ -859,7 +859,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.12"
+   "version": "3.11.1"
```
Comment (mtitov) [inline docs/source/tutorials/parsl.ipynb:862]: one more PY version

File: docs/source/tutorials/rct.ipynb [hunk 4]
```diff
@@ -1200,7 +1188,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.10"
+   "version": "3.11.1"
```
Comment (mtitov) [inline docs/source/tutorials/rct.ipynb:1191]: PY version

File: docs/source/tutorials/swift-t.ipynb [hunk 5]
```diff
@@ -35,7 +35,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.7"
+   "version": "3.11.1"
```
Comment (mtitov) [inline docs/source/tutorials/swift-t.ipynb:38]: PY version

File: docs/source/tutorials/parsl-rp.ipynb (hunk 2 above)
Comment (mturilli) [reply docs/source/tutorials/parsl-rp.ipynb:300]: Apparently, Jupiter does that automatically when you open the notebook :/

File: docs/source/tutorials.rst (hunk 1 above)
Comment (mtitov) [reply docs/source/tutorials.rst:51]: just for records as what we agreed on: keep documentation this way and use sub-directories for tutorials-container if needed

**Review by mtitov — APPROVED** [review]
LGTM!
(had an agreement with proposed changes, thus pushed them forward and to have it in the master branch)

**Pull request merged**